api/locations/models/location.model.ts

```diff
@@ -0,0 +1,26 @@
+export interface LocationModel {
```

Comment:
Is suppose to be in alphabetic order

Reply:
aggreed but this is to match the order from the API contract, for easier maintainance if they add or remove a field then we can be easier to address where it is.